index.bs

```diff
@@ -322,7 +325,7 @@ should be treated by the user agent as the user intending to grant read access t
 for the returned files. As such, at the time the promise returned by one of the [=local file system handle factories=]
 resolves, [=permission state=] for a descriptor with {{FileSystemPermissionDescriptor/handle}} set to the returned handle,
 and {{FileSystemPermissionDescriptor/mode}} set to {{"read"}}
-should be {{PermissionState/"granted"}}.
+should be {{PermissionState/granted}}.
```

Comment:
We still want quotes, just around the whole thing i.e. `"{{PermissionState/granted}}"`

Reply:
Apologies, I was considering making this change (to address https://github.com/whatwg/fs/issues/133) in this PR and then decided to leave it for a follow-up... but incorrectly undo-ed my changes 🤦 

I'll send a follow-up with these changes